langworld_db_data/adders/listed_value_adder.py

```diff
@@ -129,14 +133,16 @@ def _add_to_inventory_of_listed_values(
             )
 
             for value_index_and_line_number in value_indices_to_inventory_line_numbers:
-                if value_index_and_line_number[INDEX] == index_to_assign:
-                    line_number_of_new_value = value_index_and_line_number[LINE_NUMBER]
+                if value_index_and_line_number[KEY_FOR_FEATURE_VALUE_INDEX] == index_to_assign:
+                    line_number_of_new_value = value_index_and_line_number[
+                        KEY_FOR_FEATURE_VALUE_LINE_NUMBER
+                    ]
 
         row_with_new_value = tuple(
             [
                 {
-                    ID: id_of_new_value,
-                    FEATURE_ID: feature_id,
+                    COLUMN_FOR_PRIMARY_KEY: id_of_new_value,
+                    COLUMN_FOR_FEATURE_ID: feature_id,
```

Comment:
hm... it doesn't feel right. `id_of_new_value` is clearly not the name of a column in a table. Maybe we should split these two `COLUMN_...` constants or just use literals if each of the use cases occurs no more than twice.